eel/browsers.py

```diff
@@ -1,7 +1,16 @@
+import subprocess as sps
 import webbrowser as wbr
+
 import eel.chrome as chm
 import eel.electron as ele
-import subprocess as sps
+#import eel.edge as edg         TODO (ready to merge)
```

Comment:
What are these about?

Reply:
Just as a sort of guideline for how further browsers should be supported. You can bin it if you want